packages/ragbits-document-search/src/ragbits/document_search/ingestion/providers/unstructured/images.py

```diff
@@ -0,0 +1,111 @@
+from pathlib import Path
+from typing import Optional
+
+from PIL import Image
+from unstructured.chunking.basic import chunk_elements
+from unstructured.documents.elements import Element as UnstructuredElement
+from unstructured.documents.elements import ElementType
+
+from ragbits.core.llms.litellm import LiteLLMOptions
+from ragbits.document_search.documents.document import DocumentMeta, DocumentType
+from ragbits.document_search.documents.element import Element, ImageElement
+from ragbits.document_search.ingestion.providers.unstructured.default import UnstructuredDefaultProvider
+from ragbits.document_search.ingestion.providers.unstructured.utils import (
+    ImageDescriber,
+    crop_and_convert_to_bytes,
+    extract_image_coordinates,
+    to_text_element,
+)
+
+DEFAULT_LLM_IMAGE_SUMMARIZATION_MODEL = "gpt-4o-mini"
```

Comment:
This is not a blocker for this PR but maybe this should use our built-in `core_config.default_llm_factory` functionality? Or if not (because there is no guarantee that `default_llm_factory` returns LLM that supports images) maybe we should have a separate config option for default visual LLM?

Reply:
Created [an issue](https://github.com/deepsense-ai/ragbits/issues/140) for that, as discussed.